db/migrate/20171114232449_remove_game_mode_from_tournament.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveGameModeFromTournament < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :tournaments, :game_mode
+    remove_column :tournaments, :game_mode, :integer
```

Comment:
Okay,

- firstly: you don't need to make migrations for these changes if it's still in your branch. Nobody has created the database yet with your old migrations so you can just redo your initial `create_tournaments` or `creates_events` migrations without messing anything up.

- Secondly: If you ever have to do something like this because we already deployed something and now it's necessary to change the columns. I'd suggest changing all the affected tables in one migration (instead of three) so that the database stays in a consistent state. With rails migrations it's even possible to take the old values and move them to the new columns so that existing data doesn't get lost.